src/instructlab/model/list.py

```diff
@@ -34,32 +38,35 @@ def model_list(model_dirs: list[str], list_checkpoints: bool):
     """lists models"""
     data = []
     for directory in model_dirs:
-        for entry in Path(directory).iterdir():
-            # if file, just tally the size. This must be a GGUF.
-            if entry.is_file() and is_model_gguf(entry):
-                model, age, size = _analyze_gguf(entry)
-                data.append([model, age, size])
+        d = Path(directory)
+        if not d.is_dir():
+            continue
+        for entry in d.iterdir():
+            if is_model_gguf(entry):
+                data.append(_analyze_gguf(entry))
             elif entry.is_dir():
-                for m in _analyze_dir(entry, list_checkpoints, directory):
-                    data.append(m)
-    print_table(["Model Name", "Last Modified", "Size"], data)
+                data += list(_analyze_dir(entry, list_checkpoints, directory))
+    print_table(("Model Name", "Last Modified", "Size"), data)
 
 
-# convert_bytes_to_proper_mag takes a dir/file size in Bytes and converts it to the proper Magnitude (MiB, GiB)
-def convert_bytes_to_proper_mag(f_size: float):
-    magnitudes = ["KiB", "MiB", "GiB", "TiB"]
+def get_bytes_size_string(n_bytes: int) -> str:
+    """
+    Convert bytes size to a human readable string with the appropriate magnitude. (MiB, GiB, ...)
+    """
     magnitude = "B"
-    for mag in magnitudes:
-        if f_size >= 1024:
-            magnitude = mag
-            f_size /= 1024
-        else:
-            return f_size, magnitude
-    return f_size, magnitude
-
-
-# print_table displays the models on the system in a nicely formatted table
-def print_table(headers: list[str], data: list[list[str]]):
+    f_bytes = float(n_bytes)
+    for mag in ("KiB", "MiB", "GiB", "TiB"):
+        if f_bytes < 1024:
+            break
+        magnitude = mag
+        f_bytes /= 1024
+    return f"{.2} {magnitude}"
```

Comment:
```suggestion
    return f"{.1} {magnitude}"
```

non-blocking: I think a single digit after the decimal point is sufficient.

Reply:
I disagree, we should keep it to 2 digits - this is also how podman does it.

Reply:
I hope we agree that both `.1` and `.2` are better than unlimited (which is the current status). I picked `.2` because that's what @RobotSail used in his `data list` PR. I think it's a reasonable value. I am not a UX designer to reasonably argue for one or another. I'll keep as-is but feel free to push a follow-up changing it to something else.